### PR TITLE
Adds validation to /orgs/{id}/tags endpoint

### LINKED ISSUE #497: Add validation to add-tags to orgs
Validation for adding tags to new orgs is missing.

### PR DESCRIPTION
**Pull request by mk4111**
Closes #497

Adds validation to adding tags to org, so that only a maximum of 10 tags are added to the org

### CODE REVIEW

**Comment by mk4111** [comment]
On adding tests, I've noticed that we are using `getOrgDetails` to check whether a tag is displayed --> Problem arises because org details view displays tags attached to org and tags attached to each challenge created on behalf of org. We currently look for the name of the org on the page, which is erroneous because it could indicate that the tag exists for either the org or challenge. 

Can distinguish by HTML, but Cheerio would have been a good tool to use for this.

File: server/handlers/orgs/add-tags-view.js [hunk 1]
```diff
@@ -26,6 +26,6 @@ module.exports = function (request, reply, source, joiErr) {
 
       Hoek.assert(!pgErr, 'Database Error');
 
-      return reply.view('tags', options);
+      return reply.view('tags', options).code(error ? 401 : 200);
```
Comment (SimonLab) [inline server/handlers/orgs/add-tags-view.js]: We should change the status code for something else than 401

**Comment by JMurphyWeb** [comment]
We can always check the /org/id/edit view which includes the tags associated with that org

**Comment by mk4111** [comment]
@JMurphyWeb - yeah that's true!

**Pull request merged**